vocabularies/Org.OData.Temporal.V1.xml

```diff
@@ -137,20 +137,19 @@
         </Property>
       </ComplexType>
 
-      <ComplexType Name="Period">
-        <Annotation Term="Core.Description" String="Validity period of a &quot;delta timeslice&quot; in an action defined in this vocabulary" />
+      <ComplexType Name="TimesliceWithPeriod">
+        <Annotation Term="Core.Description" String="Delta timeslices with validity period in an action defined in this vocabulary" />
         <Annotation Term="Core.LongDescription">
-          <String>The action parameter with this type MUST be provided if the action is invoked on a collection
-          without visible data history. It MUST NOT be provided otherwise.
-          Either both `TimeSliceStart` and `TimeSliceEnd` are of type `Edm.Date`
-          or both are of type `Edm.DateTimeOffset`,
+          <String>The properties `PeriodStart` and `PeriodEnd` MUST NOT be present if the collection has visible data history.

```

Comment:
```suggestion
          <String>The properties `PeriodStart` and `PeriodEnd` MUST NOT be present if the provided type of `Timeslice` has visible data history.
```
Well, that's not a good proposal: maybe we should be silent here and instead mention this restriction on every use, where we can refer to the collection the action is bound to.